superset-frontend/src/explore/components/controls/VizTypeControl.jsx

```diff
@@ -166,15 +167,23 @@ const VizTypeControl = props => {
   const filterString = filter.toLowerCase();
 
   const filteredTypes = DEFAULT_ORDER.filter(type => registry.has(type))
-    .filter(type => !registry.get(type).isNativeFilter)
+    .filter(
+      type =>
+        isFeatureEnabled(FeatureFlag.DASHBOARD_CROSS_FILTERS) ||
```

Comment:
@simchaNielsen could you explain why we need this feature flag enabled here?

Reply:
@agatapst `FeatureFlag.DASHBOARD_NATIVE_FILTERS` feature flag filter out native filter charts from the view that responsible for creation of charts, because if they will be created as charts they will not can filter other charts, but with new feature of cross-filter functionality they will can to do it, so no need any more filter out them from this view.

Reply:
Thanks, makes sense 🙂